### PR TITLE
modified vigenere_cipher.py

### PR DESCRIPTION
**Pull request by sharyudongre**
### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by ridhishjain** [comment]
@sharyudongre some of the checks are failing, please look into it before requesting a review

File: ciphers/vigenere_cipher.py [hunk 1]
```diff
@@ -1,65 +1,75 @@
...
-    return translateMessage(key, message, "encrypt")
-
-
-def decryptMessage(key, message):
-    """
-    >>> decryptMessage('HDarji', 'Akij ra Odrjqqs Gaisq muod Mphumrs.')
-    'This is Harshil Darji from Dharmaj.'
-    """
-    return translateMessage(key, message, "decrypt")
+class cipher:
```
Comment (cclauss) [inline ciphers/vigenere_cipher.py:1]:
```suggestion
class Cipher:
```

**Review by cclauss — CHANGES_REQUESTED** [review]
How are these modifications an improvement over the current code?  I do not see the advantage.

**Comment by sharyudongre** [comment]
Thanks a lot. I will do the suggested changes.But will the checks pass
after it?

**Comment by cclauss** [comment]
You may have to make other changes to make the tests pass.

The more important question is https://github.com/TheAlgorithms/Python/pull/3265#pullrequestreview-508069858

**Comment by sharyudongre** [comment]
> How are these modifications an improvement over the current code? I do not see the advantage.

Using a class gives the code object oriented design

**Pull request closed without merging**